docs/installation.md

```diff
@@ -1,43 +1,154 @@
-# Installation
+# Installation Guide
 
-## Stable release
+## Quick Install
 
-To install auto_dev, run this command in your
-terminal:
+For users who just want to use auto_dev, install via pip:
 
 ```console
-$ pip install auto_dev
+$ pip install autonomy-dev[all]
 ```
 
-This is the preferred method to install auto_dev, as it will always install the most recent stable release.
+This installs the latest stable release with all optional dependencies.
 
-If you don't have [pip][] installed, this [Python installation guide][]
-can guide you through the process.
+### Quick Start Example
 
-## From source
+```bash
+# Create a repo & a simple webserver agent
+adev repo scaffold fun_new_hack && \
+cd fun_new_hack && \
+adev create author/cool_agent \
```

Comment:
lets ensure that publish command group is much better @xiuxiuxar can you please look at this one? 
Its the piblish command which is currently causing stuff to fall over anyhow